src/js/formats/expandable.js

```diff
@@ -18,22 +18,39 @@ export default {
         };
     },
     postRender() {
-        q('.js-expand').forEach(el => el.addEventListener('click', () => {
-            q('.js-short-content').forEach(shortContentElement => {
-                shortContentElement.classList.add('u-hidden');
-            });
-            q('.js-all-content').forEach(allContentElement => {
-                allContentElement.classList.remove('u-hidden');
-            });
-        }));
-        q('.js-collapse').forEach(el => el.addEventListener('click', () => {
-            q('.js-all-content').forEach(allContentElement => {
-                allContentElement.classList.add('u-hidden');
+        const $brexit = q('.brexit')[0];
```

Comment:
In my mind it's a bit disingenuous to put a `$` at the beginning of this as this makes me think I'm dealing with a jQuery-like wrapped set, when in reality it's just a DOM element